README.md

```diff
@@ -394,6 +394,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
 * [PyGraphviz](https://pypi.org/project/pygraphviz/) - Python interface to [Graphviz](http://www.graphviz.org/).
 * [PyQtGraph](http://www.pyqtgraph.org/) - Interactive and realtime 2D/3D/Image plotting and science/engineering widgets.
 * [Seaborn](https://github.com/mwaskom/seaborn) - Statistical data visualization using Matplotlib.
+* [Streamlit](https://streamlit.io/) - A faster way to build and share data apps.
```

Comment:
https://github.com/vinta/awesome-python/pull/1910

Reply:
Welcome to PM CRYPTO LTD, you can email our company at support@pmcryptoltd.com  to know how to start earning 7 times your capital in 48hours of trade because we are the best. 
Website: pmcryptoltd.com